ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/af-commons/v1/fieldTemplates/errorMessage.html

```diff
@@ -1,3 +1,3 @@
 <template data-sly-template.errorMessage="${@ componentId, bemBlock}">
-    <div class="${bemBlock}__errormessage"></div>
+    <div class="${bemBlock}__errormessage" id="${componentId}__errormessage" aria-live="assertive"></div>
```

Comment:
Add the same functionality for long description.

Reply:
Added